### DISCUSSION TITLE
Instance of a Country model by country code

### DISCUSSION THREAD

**Discussion by FilipKvestak** [Daniel-Ioannou/flutter_country_picker#67] [Q&A] [answered]
is it possible to get Country object by passing country code?

e.g. after picking a country, i have some data like country code, and name.
now, how do I get (later) that very same object just by country code? without opening the picker, of course

**Comment by Daniel-Ioannou** [comment, marked as answer]
@FilipKvestak To get instance of a Country use  `CountryParser.parse('US')` or `CountryParser.tryParse('US')`. For more information and more methods, please check the [documentation](https://pub.dev/documentation/country_picker/latest/country_picker/CountryParser-class.html).
